lib/stat_tracker.rb

```diff
@@ -40,4 +40,36 @@ def highest_total_score
     end
     return output.away_goals + output.home_goals
   end
+
+  def percentage_ties
+    tie_games = @game_teams.find_all do |game|
+      game.game_result.include?("TIE")
```

Comment:
I like how you solved this one, I wouldn't have thought to use `include?`. It's interesting to see different ways to do things!